### DISCUSSION TITLE
Confirmation of participating Notaries for E-Fil+ Project and service level expectations

### DISCUSSION THREAD

**Discussion by kevzak** [filecoin-project/notary-governance#579] [E-Fil+ WG]
This is a discussion about:

-  the service level commitment expectations of Notaries who will participate in the upcoming E-Fil+ Pilot
-  a confirmation of those Notaries interested in participating as an Active Notary in the E-Fil+ pilot

**Comment by kevzak** [comment]
**The following are the commitments and service level expectations as an Active Notary participating in the E-Fil+ Pilot:**

1. You are interested and willing to support the review and onboarding of large, private datasets
2. You will abide by the Code of Conduct for the E-Fil+ Pilot : [LINK](https://docs.google.com/document/d/1nWxUsoV8EfZqSD1iHUotohCNNwrBjtkclDjTgUhWyW8/edit#heading=h.n4emb0askyh4)
3. You are willing to test new processes and provide feedback during the pilot on your experience performing approvals of E-Fil+ applications
- New processes, including the use of an automated KYC system and use of a playbook

4. You are able to participate bi-weekly by:
- Providing any new feedback on all aspects of the E-Fil+ processes to @kevzak on the PL team (via survey/form)
- And/or attend Notary WG meetings and office hours

5. **E-Fil+ Pilot Key Success Metric: Notary Service Level Expectation**

-Notaries remain self directed with a goal of 48 hours (standard business hours) turnaround time when assigned an E-Fil+ LDN to review.
- As part of the pilot, we want to measure the impact of any new processes (ex: KYC automation) on the due diligence requirements (read/review of application) and understand any blockers in the time to complete appropriate approvals. 
- Application reviews can and will take longer than that the goal stated, but we want to understand those cases and evaluate as part of the pilot.

**Comment by ekil1100** [comment]
Hi @kevzak , GreaterHeat is here, willing to join the E-Fil+ Pilot Notary WG.

**Reply by kevzak** [reply]
@ekil1100 can you please confirm the Notary name that you represent?

**Reply by ekil1100** [reply]
@hexcola (GreaterHeat)

**Reply by ekil1100** [reply]
@kevzak Does it need to be Fil+ Notary to join WG?

**Reply by kevzak** [reply]
@ekil1100 you must be an active notary to be able to support the requirements for this specific WG. However, we'll have a general E-Fil+ project working group as well that is open to all to discuss the different requirements. I think that will fit you. https://github.com/filecoin-project/notary-governance/discussions/527

**Reply by Linda2049** [reply]
＠kevzak Is there any reward for the SPs?

**Reply by kevzak** [reply]
@Linda2049 do you mean DataCap reward? We are proposing this: https://github.com/filecoin-project/notary-governance/discussions/582

**Comment by kevzak** [comment]
In previous comments and discussions over the past few months, the following notaries have expressed interest in participating in the E-Fil+ Notary WG:

@MegTei
@flyworker (Charles Cao)
@MRJAVAZHAO
Danny O'Brien @dannyob
James Hoang @jameshoang
Wijnand @cryptowhizzard
Zhehao Chen @fenbushi-filecoin
@PluskitOfficial (can you confirm your name?)
@MatrixStorage (can you confirm your name?)
@davidthoms (Coffee Cloud)
@bmcnabb25 (Tinfra)
@BlockMakeronline (can you confirm your name?)

if still interested, can you please leave a comment below confirming your participation by June 29th, 2022?

**Reply by Destore2023** [reply]
Hi Kevin, You must forget to count me in. 👍

**Reply by kevzak** [reply]
Yes @swatchliu , I thought you wanted to help more so with strategy/client journey discussions, but I'm happy to add you as a notary

**Reply by NiwanDao** [reply]
Please count me in as well （Cabrina Huang）

**Reply by neogeweb3** [reply]
Count me in, thanks. (Neo Ge)

**Reply by ghost** [reply]
Hi Kevin，Count me in, thanks. (Yuan -Newwebgroup)

**Comment by cryptowhizzard** [comment]
Yup ... I am interested ( wijnand )

**Comment by bmcnabb25** [comment]
Count me in! (Brandon McNabb (Tinfra))

**Comment by MegTei** [comment]
@Megtei (Holon) Confirmed @kevzak

**Comment by davidthoms** [comment]
Hi, yes please count me in. if we can help in any way please let us know. Many clients have expressed interest in an Enterprise-level platform. We really want to help expand this side of the Fil Plus network.

**Comment by kevzak** [comment]
Confirming Active Notary participants for E-Fil+ Pilot thus far:
@MegTei 
@davidthoms 
@bmcnabb25 
@cryptowhizzard 
@xingjitansuo 
@swatchliu

**Comment by kevzak** [comment]
Awaiting a confirmation from the following Active Notaries in their participation in E-Fil+ pilot:

@flyworker (Charles Cao)
@MRJAVAZHAO
Danny O'Brien @dannyob
James Hoang @jameshoang
Zhehao Chen @Fenbushi-Filecoin
@PluskitOfficial (can you confirm your name?)
@MatrixStorage (can you confirm your name?)
@BlockMakeronline (can you confirm your name?)

please reply below if interested, thank you

**Reply by kernelogic** [reply]
I am also interested if not too late.

**Reply by liyunzhi-666** [reply]
Count me in too, thanks

**Reply by neogeweb3** [reply]
Hey @kevzak, you must forget to count me in.

**Reply by neogeweb3** [reply]
@kernelogic, I don't think there is a deadline

**Reply by kevzak** [reply]
@liyunzhi-666, apologies, I do not know all notaries by handle. Can you confirm which Notary you represent?

**Reply by liyunzhi-666** [reply]
hi @kevzak ,my organization is Tianji Studio

**Reply by PluskitOfficial** [reply]
hi, @kevzak  i'm Wuxiao~

**Reply by kevzak** [reply]
James Hoang @jamerduhgamer, apologies I had the incorrect handle.

**Reply by Fenbushi-Filecoin** [reply]
Hey @kevzak, sorry for missing the discussion. I just saw the slack group at the bi-weekly FIL+ notary meeting. Would like to participate in the E-Fil+ pilot.

**Comment by flyworker** [comment]
Yes, we are interested.

**Comment by kernelogic** [comment]
@kevzak I did not get a notification of this thread and I want to join, is it too late? I talked with @MegTei initially about supporting this.

**Reply by Tom-OriginStorage** [reply]
I guess this discussion is scattered across both Github issues and Github discussions and your conversation probably got lost

**Reply by kevzak** [reply]
@kernelogic @flyworker @neogeweb3 confirmed, thank you!

**Reply by neogeweb3** [reply]
Thanks, Kevin.

**Comment by MatrixStorage** [comment]
Hey, I'm here.DaFa  from MatrixStorage

**Comment by ghost** [comment]
Hi Kevin @kevzak ，I want to join，Count me in, thanks. (Yuan -Newwebgroup)

**Reply by kevzak** [reply]
@newwebgroup what is your slack yuan?

**Comment by derricktan23** [comment]
Hi @kevzak , Derrick from Origin Storage here, willing to join the E-Fil+ Pilot Notary WG.

**Comment by kevzak** [comment]
Thank you for confirming: @derricktan23 , @liyunzhi-666 , @newwebgroup , @BlockMakeronline @PluskitOfficial @MatrixStorage

**Comment by kevzak** [comment]
Hello - we will begin holding weekly/bi-weekly E-Fil+ Pilot Notary WG Check-in calls. 

See calendar with links:
https://calendar.google.com/calendar/u/0/embed?src=c_qpap9iqhoalpc32nmlnihhg7s0@group.calendar.google.com&ctz=America/Bahia

The first are: 
**Time: Aug 2, 2022 10:30 AM Eastern Time** 
OR
**Time: Aug 3, 2022 9:00 PM Eastern Time**

Please join either one if you can:
@derricktan23
@liyunzhi-666
@newwebgroup
@BlockMakeronline
@PluskitOfficial
@MatrixStorage
@MegTei
@davidthoms
@bmcnabb25
@cryptowhizzard
@xingjitansuo
@swatchliu
@jamerduhgamer
@kernelogic 
@flyworker
@neogeweb3
@MRJAVAZHAO

**Reply by flyworker** [reply]
Time: Aug 3, 2022 10:00 PM Eastern Time is good for me

**Reply by MRJAVAZHAO** [reply]
We will join this meeting.
Time: Aug 2, 2022 10:30 AM Eastern Time

**Reply by ghost** [reply]
If this meeting conflicts with the notary's meeting, we will choose to attend at Time: Aug 3, 2022 9:00 PM Eastern Time.

**Comment by Fatman13** [comment]
Hello! Round 3 notary @Fatman13 (same handle in Slack) from Guazi Dynamic would also like to participate. Thank you!

Do we get a Slack group or something? How do I get the Zoom link to join?

**Reply by Tom-OriginStorage** [reply]
No Slack group yet, but it would be good to hav one moving forward. The zoom link seems to be free for all and accessible through the Google calendar

**Reply by kevzak** [reply]
I sent you link to Slack subgroup @Fatman13

**Comment by kevzak** [comment]
Hello - the next Notary WG Zoom Calls will be next week, Aug 16 and 17th.

here is a link to the calendar [LINK](https://calendar.google.com/calendar/u/0/embed?src=c_qpap9iqhoalpc32nmlnihhg7s0@group.calendar.google.com&ctz=America/Bahia)

**Comment by kevzak** [comment]
Following up on a recent related open issue: https://github.com/filecoin-project/notary-governance/issues/587

This week, we held discussions about the potential number of required notaries to approve private datasets. Based on a survey to the Notary WG and SP's who have applied for the E-Fil+ Pilot: 40% of responses said to keep the process the same as today (4 Notaries required, 2 different each allocation). 60% said to have more notaries involved in approvals, up to 25% of total (in the current case that is 13).

However, there was feedback stating that private applications should be treated completely different for approvals because:
- Lead SP/Clients want to keep data private and will not want to be sharing identity/datasets or giving data samples with multiple, untrusted notaries (especially those who are also SPs).
- Instead perhaps there could be a lead Notary/governance role that meets compliance (Ex: SOC2, GDPR) to conduct the initial application review and approval who is 
- 1)not from a notary involved in the application and 
- 2)responsible for the KYC/KYB review, data sample review, and data storage plan review.

**Reply by kevzak** [reply]
There were also comments that:
- the number of notaries required should be based on size of datacap requests 
- or that perhaps there could be a randomized panel of notaries selected and the client choses from the panel who could be an approver

**Reply by kevzak** [reply]
This discussion should continue and I also want to point out that what we're discussing here regarding number of notaries required to approve overlaps with another active discussion about general Notary best practices: https://github.com/filecoin-project/notary-governance/discussions/598

**Comment by 1475Notary** [comment]
@kevzak Hi, count me in. Thanks!

**Reply by kevzak** [reply]
hello @1475Notary what is your slack?

**Comment by IPFSUnion** [comment]
@kevzak  Hi, count me in. Thanks!

**Comment by stcloudlisa** [comment]
@kevzak , hello, I joined FIL-E before, you have @ me here, but  my GitHub is blocked, there is no record of me ， can you add me again please . We have a lot of communication on slack, and we're going to have a conference call tomorrow.

**Comment by derricktan23** [comment]
@kevzak Hi, please do count me in. Thanks!

**Comment by NDLABS-Leo** [comment]
Hi @kevzak 
This is new github account for NDLabs. also I have DM u in slack. 
Please count me in. 
My address is f1yayfsv6whu3rheviucvventj3y6t542xfpb47ei 
Thank you!

**Comment by sxxfuture-official** [comment]
Hi, yes please count me in. if we can help in any way please let us know!

**Comment by JustoWang** [comment]
@kevzak Please count me in! Thanks!

**Comment by OpenGate01** [comment]
Hi @kevzak，I want to join，Count me in, thanks!

**Comment by kevzak** [comment]
Hi @OpenGate01 @NewHuoPool @sxxfuture-official thanks for your interest. This topic has shifted to a new discussion [here](https://github.com/filecoin-project/notary-governance/discussions/807) and we've updated and closed new signups for the E-Fil Multisig for now. I'll make a note to consider your involvment in the next iteration.
